about.html

```diff
@@ -12,62 +12,104 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
+    <div class="about-page">
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
 
+        <section class="tnav">
+            <header>
+            <nav>
+    
+              <img class="logo" src="img/lambda-black.png" alt="Lambda Logo.">  
+                   
+                <a href="index.html">Home</a>
+                <a href="#">About</a>
+                <a href="#">Products</a>
+                <a href="#">Blog</a>
+                <a href="#">Contact</a>
+               </nav>
+            </header>
+        </section>
+
+        <section class= "topImg">
+            <img class="jumboA" src="img/jumbo-about.png" alt="City">
+        </section>
+    
+    <section class="aboutText">
+      <div>
+    <h1>About</h1>
 
+    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+  </div>
+
+  </section> 
+  <div class="midS">
+  <section class="topAbout">
```

Comment:
For class names try to avoid naming them based on where it is on the website since things may move around especially for responsive design.